md/guest-user.md

```diff
@@ -64,31 +64,45 @@ To activate it:
     ```
-            profile|<guest_profile_name>=[case_start] 
+        [...]
+        ## ProcessPermissionRule
+        ## Let the user do get only on processes he deployed or that he supervised
+    --> GET|bpm/process=[profile|Administrator, check|org.bonitasoft.permissions.ProcessPermissionRule]
+        #POST|bpm/process=[profile|Administrator, check|org.bonitasoft.permissions.ProcessPermissionRule]
+        #PUT|bpm/process=[profile|Administrator, check|org.bonitasoft.permissions.ProcessPermissionRule]
+        #DELETE|bpm/process=[profile|Administrator, check|org.bonitasoft.permissions.ProcessPermissionRule]
+    --> GET|bpm/process/*/instantiation=[profile|Administrator, check|org.bonitasoft.permissions.ProcessInstantiationPermissionRule]
+    --> POST|bpm/process/*/instantiation=[profile|Administrator, check|org.bonitasoft.permissions.ProcessInstantiationPermissionRule]
+        [...]
     ```
-    or
-    ```
-            user|<guest_username>=[case_start] 
-    ```
-    Make sure to replace <guest_profile_name> or <guest_username> by the name of the guest profile or guest user account created in the section [Guest profile and dedicated user account](#prerequisite).
-    
 :::info 
-**Note:** If you use the [dynamic authorisation checking](rest-api-authorization#dynamic_authorization) instead of the default static authorization checking, this modification is not necessary as the rule `ProcessInstantiationPermissionRule` should already grant the access to the guest account when it is mapped to the actor initiator of the process.
+**Note:** This modification will activate the [dynamic authorisation checking](rest-api-authorization#dynamic_authorization) instead of the default static authorization checking for the following request : `GET|bpm/process`, `GET|bpm/process/*/instantiation`, `POST|bpm/process/*/instantiation`. It does not impact only the guest user access to these resources but the whole organization.  
+When a user tries to get the contract or general information of a process it will make sure he is either:
+- a member of the actor mapping of the process
+- an administrator 
+- or a manager of the process
+
+When someone tries to start a process, it will make sure he is either:
+- a member of the actor mapping defined as the initiator of the process
+- an administrator
+- or a manager of the process
 :::
 
+If in your process instantiation form you make other requests to Bonita REST API, you may need to uncomment more lines in `dynamic-permissions-checks-custom.properties` accordingly.
+
 3. Use the [platform setup tool](BonitaBPM_platform_setup) again to save your changes into the database (Eg. setup.sh/.bat push).  
    Restart Bonita server.
 
-4. If your configuration is correct a guest user should be able to start a case without being logged in first.  
+4. If your configuration is correct a guest user should be able to start a process without being logged in first.  
```

Comment:
I think we should still say "start a case" to business users and "start a process instance (case)" to technical readers.

Reply:
ok